include/onnxruntime/core/graph/graph.h

```diff
@@ -98,6 +98,12 @@ class Node {
   @remarks The graph containing this node must be resolved, otherwise nullptr will be returned. */
   const ONNX_NAMESPACE::OpSchema* Op() const noexcept;
 
+  /** Gets the opset version that the Node's operator was first defined in. 
+  @returns Opset version. If -1 the Node's operator has not been set.
+  @remarks Prefer over Op()->SinceVersion() as Op() is disabled in a minimal build
+  */
+  const int SinceVersion() const noexcept { return since_version_; }
```

Comment:
>int [](start = 8, length = 3)

Will also update the flatbuffers schema using default = -1

Reply:
not sure we want to do that. when serializing it should NEVER be -1, so we should fail serialization in that case.

Reply:
Ok, remove the default value but will change from long -> int